test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegatePerInstanceInheritedData.py

```diff
@@ -55,6 +55,11 @@ def setUpClass(cls):
 
         cls._testDir = os.path.abspath('.')
 
+    @classmethod
+    def tearDownClass(cls):
+        panel = mayaUtils.activeModelPanel()
+        cmds.modelEditor(panel, edit=True, useDefaultMaterial=False)
+
```

Comment:
If testInstanceDefaultMaterial fails the image test while useDefaultMaterial is enabled the rest of the script doesn't run to turn it off. I have to set useDefaultMaterial false in the tearDownClass so other scripts will not have it on, and also at the start of the other test runs within this script to ensure they don't have it on.